frontend/src/components/core/Sidebar/SidebarNav.test.tsx

```diff
@@ -387,16 +383,6 @@ describe("SidebarNav", () => {
     expect(props.hideParentScrollbar).not.toHaveBeenCalled()
   })
```

Comment:
This test should be removed - no longer any `mouseOver` behavior

Reply:
As mentioned on [my comment above](https://github.com/streamlit/streamlit/pull/6050#discussion_r1126425684), we decided to bring this functionality back for certain use cases, so we might want to keep the test